.github/workflows/rust.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         command: check
   linux-build:
     runs-on: "ubuntu-24.04"
-    timeout-minutes: 60
+    timeout-minutes: 75
```

Comment:
If you rebase then I think this is no longer needed.  We had accidentally made some test times larger and they should be fixed in https://github.com/lancedb/lance/commit/f5f8c14cb32c74c86a26e523e1721ec1a260b7ba

Reply:
OK, I will checkout this modify